File: Python/beecrowd_1094.py
add = [0,0,0,0]
m = int(input())
for i in range(m):
    data = list(map(str, input().split()))

    add[0] += int(data[0])
    if(data[1]=='C'):
        add[1] += int(data[0])
    elif(data[1]=='R'):
        add[2] += int(data[0])
    else: 
        add[3] += int(data[0])

a = '%'
print("Total: %d cobaias" %add[0])
print("Total de coelhos: %d" %add[1])
print("Total de ratos: %d" %add[2])
print("Total de sapos: %d" %add[3])
print("Percentual de coelhos: %.2lf %c" %(add[1]/(add[0]*1.0)*100.00, a))
print("Percentual de ratos: %.2lf %c" %(add[2]/(add[0]*1.0)*100.00, a))
print("Percentual de sapos: %.2lf %c" %(add[3]/(add[0]*1.0)*100.00, a))

File: Cpp/beecrowd_1094.cpp
#include <stdio.h>
int main() {
    int n,m,sum=0,sum1=0,sum2=0,sum3=0;
    char a;
    scanf("%d",&m);
    for(int i = m;i > 0; --i){
        scanf("%d %c",&n,&a);
        sum3 += n;
        if(a=='C'){ sum+=n; }
        else if(a=='R'){ sum1+=n; }
        else { sum2+=n; }
    }
    a = '%';
    printf("Total: %d cobaias\n",sum3);
    printf("Total de coelhos: %d\n",sum);
    printf("Total de ratos: %d\n",sum1);
    printf("Total de sapos: %d\n",sum2);
    printf("Percentual de coelhos: %.2lf %c\n", sum/(sum3*1.0)*100.00,a);
    printf("Percentual de ratos: %.2lf %c\n", sum1/(sum3*1.0)*100.00,a);
    printf("Percentual de sapos: %.2lf %c\n", sum2/(sum3*1.0)*100.00,a);
}
